crates/loam-cli/src/commands/build/build_clients.rs

```diff
@@ -102,13 +55,17 @@ impl Cmd {
         Ok(())
     }
 
+    fn loam_env(&self) -> &str {
```

Comment:
See other comment about Enum. You can add `#[default]` above a case in the enum that is the default. Or you can provide the default value in the clap args above the field.